File: level52/homework/codwars11.py
#https://www.codewars.com/kata/54ff0d1f355cfd20e60001fc/train/python
def factorial(n):
    output = 1
    if n > 12 or n < 0:
        raise ValueError
    else:
        for x in range(1, n+1):
            output *= x
        return output

File: level52/homework/codwars11.js
//https://www.codewars.com/kata/54ff0d1f355cfd20e60001fc/train/javascript
function factorial(n) {
  if (n < 0 || n > 12)
    throw RangeError();
  var f = 1;
  while (n > 1)
    f *= n--;
  return f;
}